CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 0.5.1 (Next)
 ============
 
-* Your contribution here.
+* [#204](https://github.com/ruby-grape/grape-entity/pull/204): Added ability to merge fields into hashes/root (`:merge` option for `.expose`). This also closes [#138](https://github.com/ruby-grape/grape-entity/issues/138) - [@avyy](https://github.com/avyy).
```

Comment:
Just list the numbers like [#204], [#138]: ... instead of saying "this also closes"...